### ISSUE TITLE
Backtesting Engine: Strategy fails to trigger any trades (Position remains empty)  回测结果显示空仓

### ISSUE THREAD

**Issue by xjj210130** [microsoft/qlib#2256]
## ❓ Questions and Help

我下载了20230601到20260610这段时间的A股交易信息。然后进行特征提取等操作，预测未来5到10天的股票收益，
结果预测的股票收益跑输基准股票收益，或者直接空仓。下面是我的配置信息。  

股票池custom_universe 是自己挑选表现比较好的股票。请问各位一般是什么情况呢，
qlib_init:
    provider_uri: "/root/qlib_work/qlib_data"
    region: cn

market: &market "custom_universe"
# 基准通常使用自己定义的股票，最近一月表现好的60只股票的均值
benchmark: &benchmark "SH999999" 

data_handler_config: &data_handler_config
    start_time: 2024-06-08
    end_time: 2026-06-08
    fit_start_time: 2024-07-15
    fit_end_time: 2025-06-08
    instruments: *market
    

    label: ["Ref($high, -6)/Ref($open, -1) - 1"]
    
    filter_pipe:
        - filter_type: ExpressionDFilter
          rule_expression: "Rank($amount, 0) > 0.2"
          filter_start_time: 2024-06-08
          filter_end_time: 2026-06-08
          keep: false

port_analysis_config: &port_analysis_config
    strategy:
        class: TopkDropoutStrategy
        module_path: qlib.contrib.strategy
        kwargs:
            signal: <PRED>
            topk: 15
            n_drop: 3
    backtest:
        start_time: 2025-12-08
        end_time: 2026-06-01 
        account: 100000000
        benchmark: *benchmark
        exchange_kwargs:
            limit_threshold: 0.095
            deal_price: open
            open_cost: 0.0005
            close_cost: 0.0015
            min_cost: 5

task:
    model:
        class: LGBModel
        module_path: qlib.contrib.model.gbdt
        kwargs:
            loss: rank
            colsample_bytree: 0.8879
            learning_rate: 0.2
            subsample: 0.8789
            lambda_l1: 205.6999
            lambda_l2: 580.9768
            max_depth: 8
            num_leaves: 210
            num_threads: 20
    dataset:
        class: DatasetH
        module_path: qlib.data.dataset
        kwargs:
            handler:
                class: Alpha158Plus
                module_path: my_handler
                kwargs: *data_handler_config
            segments:
                train: [2024-07-15, 2025-06-08]
                valid: [2025-06-09, 2025-12-07]
                test: [2025-12-08, 2026-06-01]
    record: 
        - class: SignalRecord
          module_path: qlib.workflow.record_temp
          kwargs: 
            model: <MODEL>
            dataset: <DATASET>
        - class: SigAnaRecord
          module_path: qlib.workflow.record_temp
          kwargs: 
            ana_long_short: False
            ann_scaler: 252
        - class: PortAnaRecord
          module_path: qlib.workflow.record_temp
          kwargs: 
            config: *port_analysis_config
